### PR TITLE
DOCS: Fixups of the migration guide and the FIPS module manual

### PR DESCRIPTION
**Pull request by levitte**
The markup needed a few touch-ups

### CODE REVIEW

**Comment by romen** [comment]
LGTM, should this be marked urgent for inclusion in the release today?

**Review by romen — APPROVED** [review]
LGTM

**Comment by h-vetinari** [comment]
I had looked at https://www.openssl.org/docs/manmaster/man7/migration_guide.html as well - aside from things that you're addressing already (like the backticks), I thought it might be useful to link the two occurrences of `INSTALL.md` to https://github.com/openssl/openssl/blob/master/INSTALL.md (or an appropriate substitute)

**Comment by levitte** [comment]
I'm unsure re urgent.  I mean, I'm not *against* it, but also don't favor rushing things in.  It's perfectly ok by me if these changes become visible by the next release instead.

**Comment by levitte** [comment]
I don't agree with linking INSTALL.md to whatever's in the master branch on github.  People should look at the INSTALL.md they got with their source download.  What happens in the master branch might not be accurate for them, because possible changes.

**Review by paulidale — APPROVED** [review]
Don't really mind making this urgent or not.

It's formatting changes not content, so not critical.

**Comment by mattcaswell** [comment]
I think this is NOT urgent.

**Comment by h-vetinari** [comment]
> 
> 
> I don't agree with linking INSTALL.md to whatever's in the master branch on github. People should look at the INSTALL.md they got with their source download. What happens in the master branch might not be accurate for them, because possible changes.

Well the page is under `manmaster`, so pointing to master is IMO quite reasonable (and basically everyone installing from source understands the potential release-vs-master divergence). Furthermore, once 3.0.0 branches, the link could/should still be pointed to INSTALL.md of that branch.

**Comment by levitte** [comment]
Ah, One More Place To Maintain!
Although, if we make it a template, a suitable perl fragment would certainly work...

Opinion, @openssl

**Comment by richsalz** [comment]
You don't need to make it a template if you write it as "INSTALL.md in the source release" (which is in the copyright on every single file :) and maybe add "The latest version can be found online at https://github.com/openssl/openssl...."

**Comment by levitte** [comment]
> ... you write it as "INSTALL.md in the source release" (which is in the copyright on every single file :)

Uhm, come again?

> Copyright xxxx-yyyy The OpenSSL Project Authors. All Rights Reserved.
> 
> Licensed under the Apache License 2.0 (the "License").  You may not use
> this file except in compliance with the License.  You can obtain a copy
> in the file LICENSE in the source distribution or at
> L<https://www.openssl.org/source/license.html>.

(cited from doc/man1/openssl-version.pod.in)

**Comment by richsalz** [comment]
I was pointing out that referencing the source code is in every single manpage.  Just trying to stave off concern that referencing the source code in the manpage is bad. :)  And snarky, yes.  Let me rephrase.

You can avoid needing a template if you write the reference like this:
```
For some additional details, see C<INSTALL.md> in the source distribution. You can find the latest copy online at
https://github.com/openssl/openssl/blob/master/INSTALL.md
```

Or you can just decide not to refer to it at all.

**Comment by levitte** [comment]
Merged

0491691342cf8fefb61de14b8edd56a937b458ac DOCS: Fixups of the migration guide and the FIPS module manual

**Comment by levitte** [comment]
Regarding the discussion of an INSTALL.md link, I'm seeing it as a matter for another PR.  Feel free!

**Comment by mattcaswell** [comment]
> Regarding the discussion of an INSTALL.md link, I'm seeing it as a matter for another PR. Feel free!

Note there is already a link to CHANGES.md on github at the top of the migration guide.
